<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>272</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout_eval">
   <item>
    <widget class="QPushButton" name="button_calculate">
     <property name="text">
      <string>Calculate Evaluation</string>
     </property>
     <property name="icon">
      <iconset theme="process-working">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="table">
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_eval_tools">
     <item>
      <widget class="QToolButton" name="view_options_button">
       <property name="text">
        <string>Set Thresholds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_evaluation_button">
       <property name="text">
        <string>Export Evaluation</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-as">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_model_button">
       <property name="text">
        <string>Update Model</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
